frontend/src/pages/org/archived-item-qa/ui/resources.ts

```diff
@@ -1,105 +1,113 @@
 import { msg } from "@lit/localize";
 import { html } from "lit";
-import { guard } from "lit/directives/guard.js";
-import { until } from "lit/directives/until.js";
-import { when } from "lit/directives/when.js";
 
 import type { ReplayData, ResourcesPayload } from "../types";
 
 import { renderSpinner } from "./spinner";
 
 import { tw } from "@/utils/tailwind";
 
-const diffImport = import("diff");
+const TOTAL = "Total";
 
 function renderDiff(
   crawlResources: ResourcesPayload["resources"],
   qaResources: ResourcesPayload["resources"],
 ) {
-  return until(
-    diffImport.then(({ diffJson }) => {
-      const diff = diffJson(crawlResources, qaResources);
-
-      const addedText = tw`bg-red-100 text-red-700`;
-      const removedText = tw`bg-red-100 text-red-100`;
-
-      return html`
-        <div
-          class=${tw`flex-1 overflow-hidden whitespace-pre-line rounded-lg border-dashed p-4 first-of-type:border-r`}
-          aria-labelledby="crawlResourcesHeading"
+  const columns = [
+    msg("Resource Type"),
+    msg("Good During Crawl"),
+    msg("Bad During Crawl"),
+    msg("Good in Replay"),
+    msg("Bad in Replay"),
+  ];
+  const rows = [
+    [
+      html`<span class=${tw`font-semibold capitalize`}
+        >${msg("All Resources")}</span
+      >`,
+      html`<span class=${tw`font-semibold`}
+        >${crawlResources[TOTAL].good.toLocaleString()}</span
+      >`,
+      html`<span class=${tw`font-semibold`}
+        >${crawlResources[TOTAL].bad.toLocaleString()}</span
+      >`,
+      html`<span
+        class="${tw`font-semibold`} ${crawlResources[TOTAL].good !==
+        qaResources[TOTAL].good
+          ? tw`text-danger`
+          : tw`text-success`}"
```

Comment:
Bit of a nit but I wonder if this and line 46 below should be `text-danger` or `text-neutral-400` to match with the categories below? The green threw me off for a second, though I figured out pretty quickly why it might be there in the All Resources line.